### ISSUE TITLE
1-24 minilint can't handle the case "({)}"

### ISSUE THREAD

**Issue by tcztzy** [ptdecker/cbasics#2]
The case "({)}" cause syntax error, but your program cannot handle this correctly.

**Comment by ptdecker** [comment]
Correct, the solution does not utilize any syntactic analysis.  The problem was solved in a way consistent with a chapter one overview lesson for the book.  A proper linter would need to be able to parse the language at hand with proper lexical analysis.  I would certainly welcome an "extras" pull request which does this.  Also, a compromise could be written that simply checks for odd numbered counters when a closing symbol is encountered which would indicate improper nesting.
